### PR TITLE
Bug 1978208: Sync dependencies for 4.8 release backports

### PR DESCRIPTION
**Pull request by PhilipGough**
* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by simonpasquier** [comment]
/bugzilla refresh
/hold

This doesn't have to be included into 4.8.0 so it can wait 4.8.z.

**Comment by simonpasquier** [comment]
It includes also the fix for https://bugzilla.redhat.com/show_bug.cgi?id=1984753

**Comment by arajkumar** [comment]
@PhilipGough, It would be great if you could sync once again to pick https://github.com/prometheus-operator/kube-prometheus/pull/1275.

**Comment by PhilipGough** [comment]
@arajkumar I spoke to @paulfantom who suggested we should sync all assets for the release branch so I did just that. Your changes are included in the latest diff.

**Review by simonpasquier — COMMENTED** [review]
/lgtm

File: assets/thanos-querier/deployment.yaml [hunk 1]
```diff
@@ -182,6 +188,9 @@ spec:
         - mountPath: /etc/kube-rbac-proxy
           name: secret-thanos-querier-kube-rbac-proxy-rules
       priorityClassName: system-cluster-critical
+      securityContext:
```
Comment (simonpasquier) [inline assets/thanos-querier/deployment.yaml:191]: this is breaking the deployment of Thanos querier. From https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-monitoring-operator/1264/pull-ci-openshift-cluster-monitoring-operator-release-4.8-e2e-agnostic-operator/1418233367471591424/artifacts/e2e-agnostic-operator/gather-extra/artifacts/deployments.json

```
"pods \"thanos-querier-6fdf64dc78-\" is forbidden: unable to validate against any security context constraint: [provider \"anyuid\": Forbidden: not usable by user or serviceaccount, provider restricted: .spec.securityContext.fsGroup: Invalid value: []int64{65534}: 65534 is not an allowed group, spec.containers[0].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[1].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[2].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[3].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[4].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], provider \"nonroot\": Forbidden: not usable by user or serviceaccount, provider \"hostmount-anyuid\": Forbidden: not usable by user or serviceaccount, provider \"machine-api-termination-handler\": Forbidden: not usable by user or serviceaccount, provider \"hostnetwork\": Forbidden: not usable by user or serviceaccount, provider \"hostaccess\": Forbidden: not usable by user or serviceaccount, provider \"node-exporter\": Forbidden: not usable by user or serviceaccount, provider \"privileged\": Forbidden: not usable by user or serviceaccount]"
```

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by ecordell** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.

*skipped*

* Score: *0.20*
* Reason: *missed cutoff today, may make it tomorrow*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Comment by ecordell** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.

*skipped*

* Score: *0.20*
* Reason: *missed cutoff today, may make it tomorrow*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Comment by ecordell** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.

*skipped*

* Score: *0.20*
* Reason: *missed cutoff for today, may make tomorrow*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Comment by ecordell** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.

*skipped*

* Score: *0.20*
* Reason: *skipping for now, may still be tagged today*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Comment by ecordell** [comment]
[patch-manager] :hourglass: This pull request was not picked by the patch manager for the current z-stream window and have to wait for the next window.

*skipped*

* Score: *0.20*
* Reason: *missed cutoff this week*

**NOTE**: This message was automatically generated, if you have questions please ask on #forum-release

**Pull request merged**